scripts/committers.rb

```diff
@@ -69,6 +69,103 @@
   puppetdb_foreman
 )
 
+@author_map = {
```

Comment:
Based on the contents of this file today in gh-pages, author_map is no longer needed.